### DISCUSSION TITLE
[FEATURE REQUEST]: Creating indexes on EMR cluster is failing

### DISCUSSION THREAD

**Discussion by mpr1** [microsoft/hyperspace#172] [Q&A]
Requesting  feature,  support of AWS - EMRFS based filesystem spark cluster 
creating indexes on EMR - EMRFS spark cluster failing with error : "HyperspaceException: Only creating index over HDFS file based scan nodes is supported."

**Comment by imback82** [comment]
@mpr1 Can you share the repro step? It seems that you didn't create a DataFrame from files, but I wanted to double-check.
